### PR TITLE
[WIP] added USDC to 0x Portal wallet

### PR DESCRIPTION
**Pull request by dave4506**
## Description

Added USDC as a default asset in the wallet experience.

TODO: add USDC into the token registry backend server. (Temporarily added the USDC asset metadata in `backend_client.ts` 

## Checklist:

-   [ ] Prefix PR title with `[WIP]` if necessary.
-   [ ] Add tests to cover changes as needed.
-   [ ] Update documentation as needed.
-   [ ] Add new entries to the relevant CHANGELOG.jsons.

### CODE REVIEW

File: packages/website/ts/components/wallet/wallet.tsx [hunk 1]
```diff
@@ -339,13 +339,15 @@ export class Wallet extends React.Component<WalletProps, WalletState> {
             tokenState.price,
             !tokenState.isLoaded,
         );
+        // console.log("primaryText", primaryText, "secondaryText", secondaryText)
```
Comment (fragosti) [inline packages/website/ts/components/wallet/wallet.tsx]: nit: should remove.

File: packages/website/ts/utils/backend_client.ts [hunk 2]
```diff
@@ -45,6 +45,13 @@ export const backendClient = {
     },
     async getTokenInfosAsync(): Promise<WebsiteBackendTokenInfo[]> {
         const result = await fetchUtils.requestAsync(utils.getBackendBaseUrl(), TOKENS_ENDPOINT);
+        // TEMPORARY: Until backend is updated with USDC, this appends the metadata for USDC
+        result.push({
+            address: '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48',
+            decimals: 6,
+            name: 'US Dollar Coin',
+            symbol: 'USDC',
+        });
```
Comment (fragosti) [inline packages/website/ts/utils/backend_client.ts]: Does this need to be in here to work, or can we not have this and just add a row to the DB?

Comment (dave4506) [reply packages/website/ts/utils/backend_client.ts]: This is a temporary fix, its better for it to be in the DB, once its in the DB this can be removed

Comment (dave4506) [reply packages/website/ts/utils/backend_client.ts]: Which specific codebase/where is this DB hosted?

**Pull request closed without merging**